testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeAlluxioCaching.java

```diff
@@ -81,9 +92,11 @@ private void testReadFromTable(String tableNameSuffix)
     /**
      * Creates a table which should contain around 6 2 MB parquet files
      */
-    private void createTestTable(String tableName)
+    private void createTestTable(String tableName, String schemaName)
```

Comment:
I expect `tableName` isn't a qualified name from these variable names. Also, we shouldn't create new schemas per table. Can we move the schema creation logic to caller side? 

```suggestion
    private void createTestTable(String schemaName, String tableName)
```